cypress/fixtures/api/users/user-before-onboarding.json

```diff
@@ -0,0 +1,46 @@
+{
+    "id": "01788ed8-6d2d-0000-ccdb-d8e17589b1ea",
+    "distinct_id": "kOyAmTq2McfCD5XvqwjKf8m6GVecm9uN0L0ypEoRN50",
+    "first_name": "Jane",
+    "email": "test@posthog.com",
+    "email_opt_in": true,
+    "anonymize_data": false,
+    "toolbar_mode": "toolbar",
+    "has_password": true,
+    "is_staff": false,
+    "is_impersonated": false,
+    "team": {
+        "id": 1,
+        "uuid": "01787e47-55d7-0000-6976-b893a03aaba8",
+        "organization": "01787e47-5558-0000-bae1-6bc513b42abe",
+        "api_token": "dtrwW0uEC7LwVx64RmZmqog_XjU0nXWuEK1d1ZpjteQ",
+        "name": "Hogflix Demo App",
+        "completed_snippet_onboarding": false,
+        "ingested_event": false,
+        "is_demo": true,
```

Comment:
prob don't want this here as there is some special handling for demo projects, and this isn't testing that

might also want to add the field on the team `has_completed_onboarding_for` since the logic also checks that field.